Content.Server/GameTicking/GameTickerCommands.cs

```diff
@@ -192,8 +204,22 @@ public void Execute(IConsoleShell shell, IPlayerSession player, string[] args)
                 shell.SendText(player, "Round has not started.");
                 return;
             }
+            else if(ticker.RunLevel == GameRunLevel.InRound)
+            {
+                string ID = args[0];
+                var positions = ticker.GetAvailablePositions();
+
+                if(positions.GetValueOrDefault(ID, 0) == 0)
```

Comment:
I'd leave a comment that the `== 0` is intentional, ie: `// n < 0 is treated as infinite`

Reply:
Addressed.